src/analyses/mutexAnalysis.ml

```diff
@@ -131,8 +131,9 @@ struct
           if not (Lockset.is_bot ctx.local) then
             let ls = Lockset.filter snd ctx.local in
             let write = match kind with
-              | `Write | `Free -> true
-              | `Read -> false
+              | Write | Free -> true
+              | Read -> false
+              | Spawn -> false (* TODO: nonsense? *)
```

Comment:
It does happen all the time though, because the third argument of `pthread_create` is a spawn access to the function's own `varinfo` (which is different from the function pointer).
The question here is, what, if anything, does a protecting lockset of a function mean. And that's different from reading the function pointer. It's like reading the function itself to execute it or something. AFAIK it doesn't really matter because we don't privatize the function itself.